shadows/framework/build.gradle

```diff
@@ -16,15 +16,54 @@ task copyNatives(type: Copy) {
     outputs.dir file("${buildDir}/resources/main")
 
     project.afterEvaluate {
-        configurations.jni.files.each { File file ->
-            def nativeJarMatch = file.name =~ /lib.*-natives-(.*)\.jar/
+        configurations.jni.files.each { File f ->
+			
+            println 'File name: ' + f.name
+			
+	    def nativeJarMatch = f.name =~ /sqlite4java-(.*)-1.0.392\.*/
+			
+	     println 'Native jar name: ' + nativeJarMatch
+			
             if (nativeJarMatch) {
-                inputs.file file
+                inputs.file f
 
+		def firstPart = nativeJarMatch[0][0]
                 def platformName = nativeJarMatch[0][1]
-                from(zipTree(file)) { rename { f -> "$platformName/$f" } }
-            }
 
+		if (f.name == 'sqlite4java-win32-x86-1.0.392.dll') {
+		
+			println 'win-x86'
+			from file(f) 
+			rename '.*.dll', 'windows-x86/sqlite4java.dll'
```

Comment:
Instead of renaming to `sqlite4java` would it be cleaner to keep the original names and modify https://github.com/robolectric/robolectric/blob/master/shadows/framework/src/main/java/org/robolectric/shadows/util/SQLiteLibraryLoader.java instead?